android/java/org/chromium/chrome/browser/vpn/BraveVpnPrefUtils.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.chromium.chrome.browser.vpn;
+
+import android.content.SharedPreferences;
+
+import org.chromium.base.ContextUtils;
+import org.chromium.chrome.R;
+
```

Comment:
Almost all of `BraveVpnPrefUtils` methods have `BraveVpn` in their names.
Maybe remove that to have like:

```
BraveVpnPrefUtils.setBraveVpnPurchaseToken("");
BraveVpnPrefUtils.setBraveVpnProductId("");
BraveVpnPrefUtils.setBraveVpnPurchaseExpiry("");
BraveVpnPrefUtils.setBraveVpnSubscriptionPurchase(false);
```
=>
```
BraveVpnPrefUtils.setPurchaseToken("");
BraveVpnPrefUtils.setProductId("");
BraveVpnPrefUtils.setPurchaseExpiry("");
BraveVpnPrefUtils.setSubscriptionPurchase(false);
```

In anyway it is not possible to invoke those in other way than through `BraveVpnPrefUtils`. wdyt?